app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    if (len(command.split()) != 3 and command.split()[0] != "cp"
+            and command.count(" ") != 2):
```

Comment:
The input validation logic is incorrect. The condition should check if the command has exactly three parts and the first part is 'cp'. The current condition uses 'and' which will not work as intended. Consider using 'or' to separate different validation checks.